src/main/kotlin/com/openlattice/organizations/HazelcastOrganizationService.kt

```diff
@@ -489,6 +490,10 @@ class HazelcastOrganizationService(
 
     fun setSmsEntitySetInformation(entitySetInformationList: Collection<SmsEntitySetInformation>) {
         phoneNumbers.setPhoneNumber(entitySetInformationList)
+
+        entitySetInformationList.groupBy { it.organizationId }.forEach { (organizationId, entitySetInfoList) ->
+            organizations.executeOnKey(organizationId, UpdateOrganizationSmsEntitySetInformationEntryProcessor(entitySetInfoList))
```

Comment:
Depending on the size of `entitySetInformationList`, you may want to replace with `.map {  .submitToKey() }.forEach { it.get() }` so that this doesn't wait for the result of each execution before sending the next out over hazelcast b/c `.executeOnKey` is synchronous